preprocessors/openstreetmap/main.py

```diff
@@ -160,5 +166,52 @@ def get_map_data():
     return response
 
 
+def unmodified_street_data(OSM_data):
+    """
+     The arguement of this function is the raw street data from
+     python overpy API, but the data cannot be accessed directly as
+     it only represents the memory address of the actual result,
+     which is also a list object. What this function does however is
+     to slightly modify it in a similar manner as would be
+     obtained directly from OVERPASS API.
+     For more information see-
+     https://python-overpy.readthedocs.io/en/latest/api.html.
```

Comment:
I have no idea what this means. Are you manually unpacking the object? Is there no way to serialize it directly? The link to the main documentation page for the library you are using does not clarify this.